src/utils/compoundApyHelpers.ts

```diff
@@ -29,6 +29,100 @@ export const tokenEarnedPerThousandDollarsCompounding = ({
   return parseFloat(interestEarned.toFixed(roundingDecimals))
 }
 
+/**
+ *
+ * @param investmentAmount - amount user wants to invest in USD
+ * @param apr - farm or pool apr as percentage. If its farm APR its only cake rewards APR without LP rewards APR
+ * @param earningTokenPrice - price of reward token
+ * @param compoundFrequency - how many compounds per 1 day, e.g. 1 = one per day, 0.142857142 - once per week
+ * @param performanceFee - performance fee as percentage
+ * @returns interest earned expressed in tokens
```

Comment:
`@returns an array of token values earned as interest, with each element representing interest earned over a different period of time` or similar might be a more accurate description?

Reply:
fixed in ada53abe3d13902799f0f00433306f778310432d